tests/functional/lib.sh

```diff
@@ -10,7 +10,7 @@ println() {
 }
 
 _sudo() {
-    if [ ${UID} = 0 ] ; then
+    if [[ ${UID} = 0 || "$HEKETI_TEST_USE_SUDO" = "no" ]]; then
```

Comment:
It's actually curious that you did it differently from how shellcheck proposed and it still passed:
```
In ./tests/functional/lib.sh line 13:
    if [ ${UID} = 0 -o "$HEKETI_TEST_USE_SUDO" = "no" ] ; then
                    ^-- SC2166: Prefer [ p ] || [ q ] as [ p -o q ] is not well defined.
```